### ISSUE TITLE
have  any play to support layer of transformer ?

### ISSUE THREAD

**Issue by pobin6** [openvinotoolkit/openvino#8702]
as the title.
I have a model, using layer of transformer, 
but the openvino not support transformer, now.

**Comment by zulkifli-halim** [comment]
hi @pobin6 For your additional information, OpenVINO supports the following Bidirectional Encoder Representations from Transformers (BERT) models, as mentioned [here](https://docs.openvinotoolkit.org/2021.4/openvino_docs_MO_DG_prepare_model_convert_model_tf_specific_Convert_BERT_From_Tensorflow.html).
·      BERT-Base, Cased
·      BERT-Base, Uncased
·      BERT-Base, Multilingual Cased
·      BERT-Base, Multilingual Uncased
·      BERT-Base, Chinese
·      BERT-Large, Cased
·      BERT-Large, Uncased

OpenVINO also provides the following seven Intel pre-trained models and one public pre-trained model:
·      [bert-large-uncased-whole-word-masking-squad-0001](https://docs.openvinotoolkit.org/2021.4/omz_models_model_bert_large_uncased_whole_word_masking_squad_0001.html)
·     [ bert-large-uncased-whole-word-masking-squad-emb-0001](https://docs.openvinotoolkit.org/2021.4/omz_models_model_bert_large_uncased_whole_word_masking_squad_emb_0001.html)
·      [bert-large-uncased-whole-word-masking-squad-int8-0001](https://docs.openvinotoolkit.org/2021.4/omz_models_model_bert_large_uncased_whole_word_masking_squad_int8_0001.html)
·      [bert-small-uncased-whole-word-masking-squad-0001](https://docs.openvinotoolkit.org/2021.4/omz_models_model_bert_small_uncased_whole_word_masking_squad_0001.html)
·      [bert-small-uncased-whole-word-masking-squad-0002](https://docs.openvinotoolkit.org/2021.4/omz_models_model_bert_small_uncased_whole_word_masking_squad_0002.html)
·      [bert-small-uncased-whole-word-masking-squad-emb-int8-0001](https://docs.openvinotoolkit.org/2021.4/omz_models_model_bert_small_uncased_whole_word_masking_squad_emb_int8_0001.html)
·      [bert-small-uncased-whole-word-masking-squad-int8-0002](https://docs.openvinotoolkit.org/2021.4/omz_models_model_bert_small_uncased_whole_word_masking_squad_int8_0002.html)
·      [bert-base-ner](https://docs.openvinotoolkit.org/2021.4/omz_models_model_bert_base_ner.html)

**Comment by zulkifli-halim** [comment]
I'm closing this issue. Feel free to re-open this issue if additional assistance is needed.

**Comment by jinfagang** [comment]
@zulkifli-halim Hi, does there any more smaller size op impelmentation such a MultiHeadAttention op in transformer? Which is aligned with pytorch calculation way
